integration-tests/chopsticks/src/tests/switchPallet/trappedAssets.test.ts

```diff
@@ -0,0 +1,205 @@
+import { test } from 'vitest'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+
+import * as PeregrineConfig from '../../network/peregrine.js'
+import * as AssetHubConfig from '../../network/assetHub.js'
+import * as RococoConfig from '../../network/rococo.js'
+import {
+	KILT,
+	getAssetSwitchParameters,
+	initialBalanceKILT,
+	initialBalanceROC,
+	keysAlice,
+	keysCharlie,
+} from '../../utils.js'
+import { peregrineContext, assethubContext, rococoContext, checkSwitchPalletInvariant } from '../index.js'
+import { createBlock, setStorage, hexAddress, getXcmMessageV4ToSendEkilt } from '../utils.js'
+import { getChildLocation, getSiblingLocationV4 } from '../../network/utils.js'
+
+/**
+ * 1. send eKILTs to peregrine while switch is paused
+ * 2. enable switch pair again
+ * 3. reclaim the assets
+ */
+test('Trapped assets', async ({ expect }) => {
+	const { checkEvents, checkSystemEvents } = withExpect(expect)
+	const switchPairParameters = getAssetSwitchParameters()
+
+	await setStorage(peregrineContext, {
+		...PeregrineConfig.createAndAssignRocs(keysCharlie.address, [
+			keysAlice.address,
+			AssetHubConfig.sovereignAccountOnSiblingChains,
+		]),
+		...PeregrineConfig.setSafeXcmVersion4(),
+		...PeregrineConfig.assignNativeTokensToAccounts(
+			[keysAlice.address, AssetHubConfig.sovereignAccountOnSiblingChains],
+			initialBalanceKILT
+		),
+		...PeregrineConfig.setSudoKey(keysAlice.address),
+	})
+
+	// pause switch pair
+	await setStorage(
+		peregrineContext,
+		PeregrineConfig.setSwitchPair(switchPairParameters, PeregrineConfig.initialPoolAccountId, 'Paused')
+	)
+
+	await setStorage(rococoContext, {
+		...RococoConfig.setSudoKey(keysAlice.address),
+		...RococoConfig.assignNativeTokensToAccounts([keysAlice.address], initialBalanceROC),
+	})
+
+	await setStorage(assethubContext, {
+		...AssetHubConfig.assignDotTokensToAccounts(
+			[keysAlice.address, PeregrineConfig.siblingSovereignAccount],
+			initialBalanceROC
+		),
+		...AssetHubConfig.createForeignAsset(keysCharlie.address, [
+			[PeregrineConfig.siblingSovereignAccount, switchPairParameters.sovereignSupply],
+			[keysAlice.address, switchPairParameters.circulatingSupply],
+		]),
+	})
+
+	// 1. send the coin and force a trap
```

Comment:
Would break the current setup. Not possible yet unfortunately

Reply:
Yeah, okay. This would require refactoring each test case, where each test case has its own describe section and setup step. According to the comment above, I assume I won't do that as part of this PR, right?

Reply:
Yes, but let's leave this conversation open for reference in the refactoring PR.